universal-login-relayer/src/config/config.ts

```diff
@@ -29,6 +25,14 @@ export interface Config {
   maxGasLimit: number;
   ipGeolocationApi: IPGeolocationApiConfig;
   httpsRedirect: boolean;
+  email: Email;
+}
+
+export interface Email {
```

Comment:
What do you think about to name it`EmailConfig`

Reply:
ok